ugsync/src/main/java/org/apache/ranger/unixusersync/config/UserGroupSyncConfig.java

```diff
@@ -1332,6 +1335,18 @@ public boolean isUserSyncNameValidationEnabled() {
 		return isUserSyncNameValidationEnabled;
 	}
 
+	public String getRegexSeparator() throws RuntimeException {
+		String val = prop.getProperty(SYNC_MAPPING_SEPARATOR);
+		if(StringUtils.isNotEmpty(val)) {
+			if (val.length() == 1)
+				return val;
+			else{
+				throw new RuntimeException("More than one character found in RegEx Separator");
+			}
+		}
+		return DEFAULT_MAPPING_SEPARATOR;
```

Comment:
We follow this pattern with in the Ranger Code for method returns, I would recommend to do the same
String ret = DEFAULT_MAPPING_SEPARATOR;
String val = prop.getProperty(SYNC_MAPPING_SEPARATOR);
if (StringUtilts.isNotEmpty(val))  {
      if (val.length() == 1) {
             ret = val;
      } else {
             throw new RuntimeException("More than one character found in RegEx Separator");
        }
 }
  return ret;
}

Reply:
Sure, will update.